pkg/storage/mvcc_history_test.go

```diff
@@ -1482,7 +1482,26 @@ func cmdScan(e *evalCtx) error {
 		opts.AllowEmpty = true
 	}
 	if e.hasArg("wholeRows") {
-		opts.WholeRowsOfSize = 10 // arbitrary, must be greater than largest column family in tests
+		for _, c := range e.td.CmdArgs {
+			if c.Key == "wholeRows" {
```

Comment:
nit: maybe you tried this already, but can't this use `e.scanArg()` and fall back to 10 when set to 0?

Either way, the test doc comment should be updated with the new argument.